heeji/최대값-만들기(1).js

```diff
@@ -0,0 +1,6 @@
+function solution(numbers) {
+    let answer = 0;
+    numbers.sort((a, b) => b - a);
+    answer = numbers[0] * numbers[1];
+    return answer;
+}
```

Comment:
```suggestion
function solution(numbers) {
    numbers.sort((a, b) => b - a);
    
    return numbers[0] * numbers[1];
}
```
- 잘푸셨습니다 👍
- 굳이 answer에 할당 하실 필요가 없습니다!!

```js
function solution(numbers) {
  return numbers.sort((a, b) => b - a)[0] * numbers[1];
}
```
- 이 방법도 참고해주세요!!!